rtloader/test/datadog_agent/datadog_agent_test.go

```diff
@@ -698,3 +698,23 @@ func TestObfuscateMongoDBString(t *testing.T) {
 	// Check for leaks
 	helpers.AssertMemoryUsage(t)
 }
+
+func TestEmitAgentTelemetry(t *testing.T) {
+	cases := []string{"counter", "histogram", "gauge"}
+	for _, tc := range cases {
+		code := fmt.Sprintf(`
+	datadog_agent.emit_agent_telemetry("test_check", "test_metric", 1.0, %s)
```

Comment:
I may be missing something, but AIUI this substitutes the metric type without quotes. Assuming the test passes, these must be defined somewhere, but I can't find them. Are these constants on Python side?

Reply:
Nope, this was just a faulty test. Since the mock verifies the arguments it should™ now catch errors like that.